spec/rubocop/cop/ezcater/feature_flag_active_spec.rb

```diff
@@ -0,0 +1,39 @@
+# encoding utf-8
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Ezcater::FeatureFlagActive, :config do
+  subject(:cop) { described_class.new(config) }
+
+  let(:msgs) { [described_class::MSG] }
```

Comment:
Should we be expecting anything about the contents of the message?  Someone could replace the constant definition with a poop emoji and this would still end up passing.

:8ball: because this is pretty nitpicky and may not be worth the trouble.